docs/create-packages/creating-a-package-dotnet-cli.md

```diff
@@ -56,80 +54,85 @@ The following example shows a simple, complete project file with these propertie
 </Project>
 ```
 
-You can also set the optional properties, such as `Title`, `PackageDescription`, and `PackageTags`, as described in [MSBuild pack targets](../reference/msbuild-targets.md#pack-target), [Controlling dependency assets](../consume-packages/package-references-in-project-files.md#controlling-dependency-assets), and [NuGet metadata properties](/dotnet/core/tools/csproj#nuget-metadata-properties).
+You can add optional properties, such as `Title`, `PackageDescription`, and `PackageTags`. You can also surface assets from dependencies directly in the package by using the `IncludeAssets` and `ExcludeAssets` attributes.
```

Comment:
I think a better description would be something along the lines of

When using the dotnet CLI, PackageReference in your project files are automatically converted dependencies for the created package. You can control what assets get included through <link to controlling` dependency assets.